docker-compose.yml

```diff
@@ -6,11 +6,12 @@ services:
     volumes:
     - ${PWD}:${PWD}
     - ${HOME}/.composer/auth.json:/root/.composer/auth.json
```

Comment:
Update to  ${HOME}/.config/composer/auth.json:/root/.config/composer/auth.json

Reply:
As there's no cross-platform solution I leave it commented out.